marklogic-data-hub/src/test/ml-modules/root/test/suites/data-hub/5/impl/security/temporal-operations/temporalOperationsTest.sjs

```diff
@@ -19,14 +19,37 @@ const setLsqtUsingAForbiddenUser = (roles) => {
   });
 }
 
-const setLsqtUsingAPermittedUser = (roles) => {
+const setLsqtAutomationUsingAForbiddenUser = (roles) => {
+  hubTest.runWithRolesAndPrivileges(roles, [], function () {
+    xdmp.invokeFunction(function () {
+      try {
+        temporal.setLsqtAutomation("kool", true, 5000);
+        assertions.push(test.fail('Exception not thrown when attempting to set LSQT Automation using a forbidden user'));
```

Comment:
One note, no need to change - I would just do "throw Error(message)" here, less code.